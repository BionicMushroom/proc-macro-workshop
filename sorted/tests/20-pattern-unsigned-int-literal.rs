#[sorted::check]
fn main() {
    let x = 15u128;

    #[sorted]
    match x {
        0 => println!("0"),
        340_282_366_920_938_463_463_374_607_431_768_211_455 => {
            println!("340_282_366_920_938_463_463_374_607_431_768_211_455")
        }
        1 => println!("1"),
        _ => println!("other"),
    }
}
