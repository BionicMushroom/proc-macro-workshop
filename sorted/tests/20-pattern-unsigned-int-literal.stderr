error: 1 should sort before 340_282_366_920_938_463_463_374_607_431_768_211_455
  --> tests/20-pattern-unsigned-int-literal.rs:11:9
   |
11 |         1 => println!("1"),
   |         ^
